src/Network/NWProtocolUdpOptions.cs

```diff
@@ -0,0 +1,43 @@
+//
+// NWProtocolTls: Bindings the Netowrk nw_protocol_options API focus on Udp options.
+//
+// Authors:
+//   Manuel de la Pena <mandel@microsoft.com>
+//
+// Copyrigh 2019 Microsoft Inc
+//
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+using Security;
+using OS_nw_protocol_definition=System.IntPtr;
+using IntPtr=System.IntPtr;
+
+namespace Network {
+
+	[TV (12,0), Mac (10,14), iOS (12,0), Watch (6,0)]
+	public class NWProtocolUdpOptions : NWProtocolOptions {
+		bool preferNoChecksum = false;
```

Comment:
same issue

and https://github.com/xamarin/xamarin-macios/pull/7196/files#diff-2f2e76e5bddde26c68898ef0220c89adR84 needs to be updated